pom.xml

```diff
@@ -83,7 +83,7 @@
   <com.google.dagger.version>2.45</com.google.dagger.version>
   <com.google.dagger.compiler.version>2.45</com.google.dagger.compiler.version>
 
-  <io.cryostat.core.version>2.21.0</io.cryostat.core.version>
+  <io.cryostat.core.version>2.22.0</io.cryostat.core.version>
```

Comment:
Requires a -core bump?

Reply:
yep updated from `2.21.0` to `2.22.0`. Just waiting for -core to release `2.22.0` version :))

Reply:
`2.21.1` should be available now.

Reply:
Opps thanks! Updated now!

Reply:
No worries, I just did that earlier this morning.